docs/resources/covid-19.md

```diff
@@ -1,44 +1,182 @@
+
+and even more, **safety information** can be retrieved for the destination you would like to visit so that you will be aware of where you are heading to. 
 
 ## Search by an area
-the [Travel Restrictions API](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/travel-restrictions) can be searched by a country, city or region. 
- 
+
+There are 2 APIs to achieve the information 
+1. [Amadeus Travel Restrictions API](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/travel-restrictions) for covid-19 related information 
+2. [Amadeus Safety Place API](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/safe-place) for Safety information
+
+[Travel Restrictions API](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/travel-restrictions) can be searched **by a country, city, or region**. 
+
 ```bash
-curl https://test.api.amadeus.com/v1/duty-of-care/diseases/covid19-area-report?countryCode=KR&cityCode=SEL
+curl https://test.api.amadeus.com/v1/duty-of-care/diseases/covid19-area-report?countryCode=FR&cityCode=PAR
 ```
-
 !!! information
     Country code(2 letters code) is defined in [ISO 3166](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) and city code(3 letters code) is defined in [IATA](https://www.iata.org/en/publications/directories/code-search/), which you can also use [Amadeus Airport & City Search API](https://developers.amadeus.com/self-service/category/air) to get.
 
 
-## Get Covid-19 statistics 
-the result provides Covid-19 Statstics such as total cases, active cases, and current rate of infection per 100,000 
+[Safety Place API](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/safe-place) can be searched by a given location with **latitude, longitude and radius**, or **Square information** (North, West, South, and east) or **by id**.
+
+```bash
+curl https://test.api.amadeus.com/v1/safety/safety-rated-locations?latitude=48.856614&longitude=2.3522219&radius=2
+```
+
+```bash
+curl https://test.api.amadeus.com/v1/safety/safety-rated-locations/Q930400878
+```
+
+!!! information 
+    the id is given by your first query which has been searched by others. the search by id was designed to target a specific area that you want to focus on.
+
+
+
+
+## Get Information from the responses
+Let's go through some highlights information you get from both Travel Restriction API and Safety Place API. 
+
+!!!information
+   Don't forget that Amadeus for Developers provides a `Test Environment` with [limited data collections](https://amadeus4dev.github.io/developer-guides/guides/api-data-collection/). 
+   - the data source of travel restrictions API is from [Riskline](https://riskline.com/), they have been sourced from local governments and media. The quantity of information provided may vary from country to country. 
```

Comment:
Travel Restrictions API 
"they have been sourced from" -> I think the data is singular always, so it should be "the data has been sourced"